lib/libc/math/lib_erfc.c

```diff
@@ -0,0 +1,39 @@
+/****************************************************************************
```

Comment:
We added samsung license to all files in lib directory. Could you add it with 2023 year for new files you added?

/****************************************************************************
 *
 * Copyright 2023 Samsung Electronics All Rights Reserved.

Reply:
Done, please check.

Reply:
You changed Nuttx license to Samsung one. But if you get this from Nuttx, you must keep Nuttx license. And then you should add the Samsung license at the top.
````
samsung license
nuttx license
codes
```

Reply:
Correction done.